lib/tether/netfilter/netfilter_linux.go

```diff
@@ -158,7 +160,7 @@ func (r *Rule) args() ([]string, error) {
 	return args, nil
 }
 
-func iptables(ctx context.Context, args []string) error {
+func iptables(ctx context.Context, args []string) (tether.UtilityFn, error) {
 	args = append(args, "-w")
```

Comment:
Now that the iptables calls are being executed sequentially, the flag to wait for the xtables lock is no longer necessary.